_security/configuration/demo-configuration.md

```diff
@@ -27,7 +27,7 @@ The demo configuration is automatically called as part of the setup for each sup
 Use the following steps to set up the Security plugin using Docker:
 
 1. Download [docker-compose.yml](https://opensearch.org/downloads.html).
-2. In the `docker-compose.yml` file, set `DISABLE_SECURITY_PLUGIN` to `false`.
+2. In the `docker-compose.yml` file, set environment variable `DISABLE_SECURITY_PLUGIN` to `true`.
```

Comment:
This section should be changed to `if user want to disable security plugin then set to true` or similar.

More details can be seen here:
https://github.com/opensearch-project/opensearch-build/tree/main/docker/release#disable-security-plugin-security-dashboards-plugin-security-demo-configurations-and-related-configurations

We have security always enable by default so no need to specifically make it false.
And this commit of making it to true will just disable the security by default, which is not the recommended way for docker.

Reply:
The correct way to fix this is just changing `Use the following steps to set up the Security plugin using Docker` to something like `Use the following steps to disable Security plugin when using Docker`, or just directly link to the readme file for docker release in build repo:
https://github.com/opensearch-project/opensearch-build/blob/main/docker/release/README.md#disable-security-plugin-security-dashboards-plugin-security-demo-configurations-and-related-configurations

We do not want to recommend users to disable security unless they specifically want to do so.

Reply:
@peterzhuamazon - Let me know if this works. I took that line out of the process steps and moved it below.